BootloaderCorePkg/Stage1A/Stage1A.c

```diff
@@ -369,6 +411,9 @@ ContinueFunc (
 
   // Jump into Stage 1B entry
   if (StageBase != 0) {
+    ImageContext.ImageAddress = (PHYSICAL_ADDRESS)(UINTN) GET_STAGE_MODULE_BASE (StageBase);
```

Comment:
Maybe a good idea to add a StageEntry hook point to call these functions.   Then the code will be simpler.

Reply:
Okay, let me make a common interface.